### PR TITLE
446 - Added crosstable sums

### LINKED ISSUE #446: bring back crosstable sums
Commented code in renderMatch method.  I think it could be moved to the CrossTable component, since that's the only place we use it.

### PR DESCRIPTION
**Pull request by coreymaher**
Fixes #446

### CODE REVIEW

File: src/components/Match/CrossTable.jsx [hunk 1]
```diff
@@ -48,7 +53,95 @@ const CrossTable = ({
...
           </TableRowColumn>);
         })}
+        {(() => {
+          const hero1 = heroes[player.hero_id] || {};
+          let ptotal1 = 0;
+          let ptotal2 = 0;
+
+          match.players.slice(0, match.players.length / 2).forEach((player2) => {
+            const hero2 = heroes[player2.hero_id];
+            ptotal1 += (player[field1][hero2.name] || 0);
```
Comment (howardchung) [inline src/components/Match/CrossTable.jsx]: will this crash if !player[field1]?

**Comment by howardchung** [comment]
Can you also remove the commented out chunk of code in Match/renderMatch?

**Comment by coreymaher** [comment]
Sure, I'll make the requested changes after work

**Comment by coreymaher** [comment]
@howardchung  I've made some changes. Take a look when you have a chance.

**Pull request merged**